### PR TITLE
decoupled import/export functions

### PR DESCRIPTION
**Pull request by akp111**

### CODE REVIEW

File: src/constants/screen.ts [hunk 1]
```diff
@@ -10,8 +10,10 @@ export const taskConstants: ScreenConstantsInterface = {
   'Check on-chain balance': "balance",
   'Get network info': "network",
   'Get validator info': "validators",
-  'Move assets from C-chain to P-chain': 'CP',
-  'Move assets from P-chain to C-chain': "PC",
+  'Export assets from C-chain to P-chain': 'CP',
```
Comment (avisingh2) [inline src/constants/screen.ts:13]: be consistent  use single quotes as per the prettierc.yml file

Comment (avisingh2) [reply src/constants/screen.ts:13]: How do you identify the difference between export and import where you use pc and pc for both import and export.

**Comment by avisingh2** [comment]
@akp111 can you resolve and address the review comments

**Comment by akp111** [comment]
Fixed via `waitFinalize`

**Pull request closed without merging**